tests/src/Unit/Cache/Context/OgCacheContextTestBase.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Drupal\Tests\og\Unit\Cache\Context;
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\og\OgContextInterface;
+use Drupal\Tests\UnitTestCase;
+
+/**
+ * Base class for testing cache context services.
+ */
+abstract class OgCacheContextTestBase extends UnitTestCase {
+
+  /**
+   * The mocked OG context service.
+   *
+   * @var \Drupal\og\OgContextInterface|\Prophecy\Prophecy\ObjectProphecy
+   */
+  protected $ogContext;
+
+  /**
+   * A mocked group entity.
+   *
+   * @var \Drupal\Core\Entity\EntityInterface|\Prophecy\Prophecy\ObjectProphecy
+   */
+  protected $group;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp() {
+    parent::setUp();
+
+    $this->ogContext = $this->prophesize(OgContextInterface::class);
+    $this->group = $this->prophesize(EntityInterface::class);
+  }
+
+  /**
+   * Tests the result of the cache context service with active context objects.
+   *
+   * This is the 'normal' test case. The service will be able to retrieve data
```

Comment:
I mean that it is testing the regular process, how it is intended to be used. I'll update the wording.